.github/workflows/changelog.yml

```diff
@@ -188,3 +188,6 @@ jobs:
           changeLogPath: xtensa-lx-rt/CHANGELOG.md
           skipLabels: "skip-changelog"
           missingUpdateErrorMessage: "Please add a changelog entry in the xtensa-lx-rt/CHANGELOG.md file."
+
+      - name: Changelog format check
```

Comment:
We need to make sure the skip change log label can skip running this job

Reply:
Not sure why, I don't want us to be able to break the changelog formatting with that label.

Reply:
Ah my bad, I got confused by https://github.com/esp-rs/esp-hal/pull/3478#issuecomment-2862652422 and though this was also doing the changelog check, but it's just formatting.